starters/solidjs-tailwind/README.md

```diff
@@ -35,6 +35,8 @@ This starter kit features the most two hot frameworks in JavaScript and CSS echo
 
 ## Installation
 
+This README is written with pnpm in mind but you can use other manager runner 
```

Comment:
Please add the possible package manager. And put you can give an example too. Like npm run dev, or pnpm run dev or yarn dev